eslint.config.mjs

```diff
@@ -0,0 +1,22 @@
+import bpmnIoPlugin from 'eslint-plugin-bpmn-io';
+
+export default [
+  ...bpmnIoPlugin.configs.browser.map((config) => {
+    return {
+      ...config,
+      files: [ 'lib/**/*.js' ]
+    };
+  }),
+  ...bpmnIoPlugin.configs.node.map((config) => {
+    return {
+      ...config,
+      files: [ '*.js', '*.mjs', 'test/suite.js' ]
+    };
```

Comment:
Let's instead add a `/* global require */` in test / suite, as this "contains" the spill of it.

Reply:
When removing `'test/suite.js'` the file will not be linted at all for node-specific things so `/* global require */` would have no effect.

Reply:
...

See https://github.com/bpmn-io/diagram-js-direct-editing/pull/48/commits/3fad959d91dbaf8cfd25d42a621f69eb91a45935 :upside_down_face: